docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,790 @@
+    "\n",
+    "from plasmapy.dispersion.analytical.two_fluid_ import two_fluid\n",
+    "from plasmapy.dispersion.numerical.hollweg_ import hollweg\n",
+    "from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency, wc_, wp_\n",
+    "from plasmapy.formulary.lengths import inertial_length\n",
+    "from plasmapy.formulary.speeds import Alfven_speed, cs_, ion_sound_speed, va_\n",
+    "from plasmapy.particles import Particle\n",
+    "from plasmapy.utils.exceptions import PhysicsWarning\n",
+    "\n",
+    "warnings.filterwarnings(\n",
+    "    action=\"ignore\",\n",
+    "    category=PhysicsWarning,\n",
+    "    module=\"plasmapy.dispersion.numerical.hollweg_\",\n",
+    ")\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = [10.5, 0.56 * 10.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Wave propogating at nearly 90 degrees\n",
+    "\n",
+    "Below we define the required parameters to compute the wave frequencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define input parameters\n",
+    "inputs = {\n",
+    "    \"k\": np.logspace(-7, -2, 300) * u.rad / u.m,\n",
+    "    \"theta\": 89 * u.deg,\n",
+    "    \"n_i\": 5 * u.cm ** -3,\n",
+    "    \"B\": 1.10232e-8 * u.T,\n",
+    "    \"T_e\": 1.6e6 * u.K,\n",
+    "    \"T_i\": 4.0e5 * u.K,\n",
+    "    \"ion\": Particle(\"p+\"),\n",
+    "}\n",
+    "\n",
+    "# a few useful plasma parameters\n",
+    "params = {\n",
+    "    \"n_e\": inputs[\"n_i\"] * abs(inputs[\"ion\"].charge_number),\n",
+    "    \"cs\": ion_sound_speed(\n",
+    "        inputs[\"T_e\"],\n",
+    "        inputs[\"T_i\"],\n",
+    "        inputs[\"ion\"],\n",
+    "    ),\n",
+    "    \"va\": Alfven_speed(\n",
+    "        inputs[\"B\"],\n",
+    "        inputs[\"n_i\"],\n",
+    "        ion=inputs[\"ion\"],\n",
+    "    ),\n",
+    "    \"wci\": gyrofrequency(inputs[\"B\"], inputs[\"ion\"]),\n",
+    "}\n",
+    "params[\"lpe\"] = inertial_length(params[\"n_e\"], \"e-\")\n",
+    "params[\"wpe\"] = plasma_frequency(params[\"n_e\"], \"e-\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The computed wave frequencies (rad/s) are returned in a dictionary with the keys representing\n",
+    "wave modes and the values being an Astropy [Quantity](https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity). Since our inputs were a 1D arrays of\n",
+    "of $k$'s, the computed wave frequencies will be a 1D array of size equal to the size of the $k$ array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# compute\n",
+    "omegas1 = hollweg(**inputs)\n",
+    "omegas2 = two_fluid(**inputs)\n",
+    "np.set_printoptions(precision=4, threshold=20)\n",
```

Comment:
Put this up in the import/configuration cell above, just so we can keep this cell all about the omegas.